### ISSUE TITLE
Fix Accessibility Issues on the "Updating the app stores" page of the Documentation Site

### ISSUE THREAD

**Issue by brea11y** [department-of-veterans-affairs/va-mobile-app#8463]
During the accessibility audit of the documentation site, issues were discovered on the "Updating the app stores" page of the documentation site. Those issues can be found [in the Google sheet](https://docs.google.com/spreadsheets/d/1agEqXg74Cj2K4mEw719qwQdqUrQL7ipSOEPT1IiDCFI/edit?usp=sharing) and in the list below.

 - [ ] **Page:** Updating the app stores
    -  **Severity:** Serious
    -  **Issue:** The table is not fully accessible
   - **Recommendation:** Revise the table to be more accessible to screenreader users (i.e. ensure the top row are labeled as <th>, etc.)
 - [ ] **Page:** Updating the app stores
   -  **Severity:** Moderate
   -  **Issue:** Skipped heading level
   -  **Recommendation:** "Update content" should be an H2. It is currently an H3. Review heading levels to ensure that none are skipped.
